README.md

```diff
@@ -1,9 +1,10 @@
-player      | order | points
------------ | ----- | ------
-akvadrako   | Ba    | -2
-RamonRB     | Be    | 11
-rknegjens   | K     | 0
-roarandrock | W     | 0
+player      | order | points | VK
+----------- | ----- | ------ | -- 
+akvadrako   | Ba    | -2     | 0
+RamonRB     | Be    | 11     | 0
+rknegjens   | K     | 0      | 0
```

Comment:
You could also adjust all the points and VK levels here, like if the PR is accepted.